### PR TITLE
Eddies change

### PR DESCRIPTION
**Pull request by ATOPesi**
Added "$USER" in path for push_update script.  Added some other logic in "find_load_proc()" it work with both vtllc and sbalc labs.  Also added print statement code to print timestamp before kicking off test.

### CODE REVIEW

File: load_post_run.py [hunk 1]
```diff
@@ -85,23 +86,24 @@ def find_load_proc():
...
                 load_proc = i    
                 #print("mcpp " + i)    
             elif "fdps" in i:
                 fdps = i
-    
+    print("YOOO2: " + load_proc)    
     #remove the lines.remove(i). it was skipping lines and it missed FDP for my ZOA config. no need to remove lines, since python will increment on its own. 
     #also added error handling with empty values of load_proc and fdps
-    if load_proc == "" or fdps == "":
+    if load_proc == "" and fdps == "":
```
Comment (gacayana) [inline load_post_run.py:106]: I think keeping or would be better. Should not proceed if it's missing one of the variables, not both.

File: load_post_run.py [hunk 2]
```diff
@@ -85,23 +86,24 @@ def find_load_proc():
...
         for i in lines:
             print("For loop: " + i)    
             #if statement finds the fdps nad mcpp in the config file if there is one and assigns the mcpp string to the 'load_proc' variable and the fdps to the 'fdps' variable
             if len(lines) == 1:
                 load_proc = i
+                print("YOOO: " + load_proc)
                 break
             #needs to be more specific for instances that mutliple MCPs. for instance, ZOA has mcppa101 and mcppa102. needs to be mcppa101 since mcppa102 is the PPP position.
-            elif i.startswith('mcp') and i.endswith('1'):
+            elif i.startswith('mcp') and i.endswith('1') or 'mcppa101' in i or 'mcppb601' in i:
```
Comment (gacayana) [inline load_post_run.py:98]: Sooo I realized that for some reason, I didn't see this error for ZNY DIL, when you use half_NYa2.config and half_NYb2.config, when your MCP is set to mcppa102/ b602. I'm going to figure out how to fix this, but in the meantime, this wouldn't help. I think we should leave this as is for now.

**Review by gacayana — APPROVED** [review]

**Review by gacayana — APPROVED** [review]

File: load_post_run.py (hunk 1 above)
Comment (ATOPesi) [reply load_post_run.py:106]: This was a change that was made to fix the single box case.  Discussed another avenue for error handling in a different line.

File: load_post_run.py (hunk 2 above)
Comment (ATOPesi) [reply load_post_run.py:98]: This will be fixed in esi6.

**Pull request merged**